### PR TITLE
Fix transformation issue

### PR DESCRIPTION
**Pull request by mathieuroques**
Now numerical values are supported

### CODE REVIEW

**Comment by mathieuroques** [comment]
I struggle to add drop-down selector to column_name and default value to mad_num
image

**Comment by mathieuroques** [comment]
Not sure we need the save transformation CTA anymore as we can add it directly in the default config

**Comment by mathieuroques** [comment]
@rabah-khalek It should be ok now

File: python-client/giskard/ml_worker/testing/functions/transformation.py [hunk 1]
```diff
@@ -139,3 +140,17 @@ def change_writing_style(
...
     x.at[index, column_name] = chain_rewrite.run({"text": x.at[index, column_name], "style": style})
     return x
+
+
+@transformation_function(name="MAD Increment", tags=["num"], row_level=False)
+def mad_transformation(
+    data: pd.DataFrame, column_name: str, factor: float = 3, value_added: float = None
+) -> pd.DataFrame:
+    """
+    Add 3 times the MAD value to the column
```
Comment (rabah-khalek) [inline python-client/giskard/ml_worker/testing/functions/transformation.py:150]: This should rather be:
```py
"""
Add 3 times the value_added to the column, or if unavailable, add 3 times the MAD value.
"""
```

**Review by rabah-khalek — APPROVED** [review]
I just left you a small comment on the docstring of `mad_transformation`

File: python-client/giskard/push/utils.py [hunk 2]
```diff
@@ -31,3 +31,10 @@ def bins_count(model, dataframe):  # done at the beggining
         flag = {value: count < ratio for value, count in counts.items()}
         value_counts[column] = {"value_counts": counts, "nunique": nunique, "ratio": ratio, "flag": flag}
     return value_counts
+
+
+def compute_mad(x):
```
Comment (rabah-khalek) [inline python-client/giskard/push/utils.py:36]: I would also suggest to use the scipy stats method, it's fast and more transparent. The only down side could be the version compatibility, but as long as it works with the scipy that we ship, should be fine. 
Screenshot 2023-07-24 at 16 35 10

**Review by rabah-khalek — CHANGES_REQUESTED** [review]

**Pull request merged**